recipe-client-addon/lib/NormandyDriver.jsm

```diff
@@ -31,8 +31,11 @@ this.NormandyDriver = function(sandboxManager) {
   }
   const {sandbox} = sandboxManager;
 
+  Storage.seedDurability(sandbox);
```

Comment:
Unfortunately, we [create one driver instances per recipe we run](https://github.com/mozilla/normandy/blob/b589f6c3eaa37f73025b87976cc08b78d9f94ca6/recipe-client-addon/lib/RecipeRunner.jsm#L165), so this will count only the first recipe run the first time as non-durable. It would probably be good to call seedDurability from the context of RecipeRunner, so that it only runs once.